src/vivarium_csu_hypertension_sdc/components/hypertension_medication.py

```diff
@@ -125,7 +127,7 @@ def setup(self, builder):
         self.efficacy_data = data_transformations.load_efficacy_data(builder).reset_index()
 
         self.drug_efficacy = pd.Series(index=pd.MultiIndex(levels=[[], [], []],
-                                                                 labels=[[], [], []],
+                                                                 codes=[[], [], []],
```

Comment:
this and the change below in utilities are not substantive changes - I just got annoyed with the pandas future warnings showing up when I was running sims to test so I went and tracked these down and fixed them